cds_ils/migrator/utils.py

```diff
@@ -154,3 +166,130 @@ def clean_created_by_field(record):
         record["created_by"] = {"type": "script", "value": "migration"}
 
     return record
+
+
+def get_acq_ill_notes(record):
+    """Extract notes for acquisition and ILL records."""
+    # NOTE: library notes are usually empty dict stringified '{}'
+    result = ""
+    cost = record.get("cost")
+    if cost:
+        result += "cost: {0}\n\n".format(cost)
+
+    item_info = record.get("item_info")
+    if item_info:
+        result = "item info: {0}\n\n".format(item_info.strip())
+
+    comments = record.get("borrower_comments")
+    if comments:
+        result = "borrower comments: {0}\n\n".format(comments.strip())
+
+    notes = record.get("library_notes")
+    if not notes:
+        return result
+
+    try:
+        if json.loads(notes):
+            result += "library notes: {0}\n".format(notes.strip())
+    except json.decoder.JSONDecodeError:
+        # At this point if we failed to parse json, we have notes as string
+        result += "library notes: {0}\n".format(notes.strip())
```

Comment:
looks like there is no difference between the 2 and the try/except is not doing much, so just add them to the result?